assets/css/abstracts/_colors.scss

```diff
@@ -47,5 +47,9 @@ $core-orange: #ca4a1f;
 // @todo: replace those colors with the correct values once we settle on a design system palette.
 $gray-10: #c3c4c7;
 $gray-20: #a7aaad;
+$gray-50: #646970;
```

Comment:
I just found out this color is not used anywhere, right? Should it be removed or is there any reason to add it here?

Reply:
This was removed by mistake in a refactor, it is now restored in 981b072